modules/ROOT/pages/apikit-workflow-convert-existing.adoc

```diff
@@ -1,104 +1,239 @@
-
-== Map RAML Resources to Actions
-
-As part of the backend-first design process, you map RAML resources to actions as shown in the following procedure. This procedure assumes you have generated backend flows.
-
-To map resources to actions:
-
-. Click the APIkit Router to open the *Properties Editor*. In *Router configuration*, click image:add-16x16.png["Add button."].
+The *Global Element Properties* wizard appears.
+. In *API Definition*, select the path `api/api.raml`.
+. Accept the configuration.
+. Delete the HTTP listener from `GET-Users-FLOW`.
+. Delete the HTTP listener from `GET-User-By-Id-FLOW`.
+
+[source,xml,linenums]
+----
+<mule>
+    <apikit:config outboundHeadersMapName="outboundHeadersMapName" httpStatusVarName="httpStatus" name="Router" api="api/api.raml" />
+    <http:listener-config name="HTTP_Listener_config">
+		<http:listener-connection host="0.0.0.0" port="8081" />
+	</http:listener-config>
+	<db:config name="Database_Config">
+		<db:my-sql-connection host="localhost" port="3306" user="admin" password="admin" database="usersdb" />
+	</db:config>
+    <flow name="apiFlow">
+		<http:listener config-ref="HTTP_Listener_config" path="/api/*"/>
+		<apikit:router config-ref="Router"/>
+	</flow>
+	<flow name="GET-Users-FLOW">
+		<db:select config-ref="Database_Config">
+			<db:sql ><![CDATA[SELECT * FROM USERS]]></db:sql>
+		</db:select>
+	</flow>
+	<flow name="GET-User-By-Id-FLOW">
+		<db:select config-ref="Database_Config" >
+			<db:sql ><![CDATA[SELECT * FROM USERS WHERE :userIdParameter == userID]]></db:sql>
+			<db:input-parameters ><![CDATA[#[{userIdParameter: attributes.queryParams.id}]]]></db:input-parameters>
+		</db:select>
+	</flow>
+</mule>
+----
+
+=== Step 3: Map API Resources-Actions to Flows
+
+By default, each APIkit flow adheres to a naming convention for routing purposes, characterized by the following pattern:
+
+`action:\resource[:content-type][:config-name]`
+
+[options="header"]
+|===
+| Name | Description
+| `action` | HTTP method
+| `resource` | HTTP resource
+| `content-type` | Payload content-type (optional)
+| `config-name` | APIkit router configuration name (optional)
+|===
+
+To retain the original flow names and map each API resource-action to them, override this behavior by following these steps:
+
+. Select *APIkit Router* to open the *Properties Editor*. In *Router configuration*, click *Edit*.
 +
 The *Global Element Properties* wizard appears.
 +
-. Enter the relative path to the RAML file you created within Studio, for example: src/main/resources/api/api.raml. If you browse, the file name alone is used and that isn't sufficient.
-. In Mappings, click image:add-16x16.png["Add button"] to create a new mapping.
+. In *Mappings*, click `Add` to create a new mapping configuration.
 +
 The *New Mapping* dialog appears.
 . Use the drop-down to map the resources to actions.
 +
-* In the Resource drop-down, select `/users`.
+* In the *Resource* drop-down, select `/users`.
```

Comment:
@dhirsch1, do you mean that the "drop-down" should be deleted so the text reads:
 
The *New Mapping* dialog appears.
. Map the resources to actions.
+
* In Resource, select `/users`.

This change should be done globally.
Did I get it right?

Reply:
Yes, but it should be "In Resources, `select /users.` (with Resources in bold)

From our style guide:

**UI elements**
When directing your audience to use the UI, use the label of each UI element exactly as it appears onscreen (excluding ending punctuation), even if the label violates our usual presentation and style guidelines.

Consider referring to UI elements by label rather than element type:

Before
. Select the Upload all my files now checkbox and click the Next button.

After
. Select Upload all my files now and click Next.

Reply:
OK! Let me tell you that "select" is not part of the resource. "Select" is the transitive verb and `/users` is the object.
Thanks!